### ISSUE TITLE
First feedBase demo, Q&A

### ISSUE THREAD

**Issue by scripting** [scripting/Scripting-News#71]
I just posted a video demo of feedBase. Have a look, read the post, and if you have questions post them here. Let's have fun!

**Comment by FTWynn** [comment]
Interesting concept as always Dave.

Are there any plans to support grouping feeds together via tags or something? I uploaded an opml file (exported from feedly) with several groups that were all flattened out on import to Feedbase. It would be great to keep the comics together, the news sites together, etc, so I wouldn't have to recreate those groups in all the downstream apps.

[ftwynn.opml.txt](https://github.com/scripting/Scripting-News/files/1853563/ftwynn.opml.txt)

**Comment by scripting** [comment]
Yes. For example I want to publish the podcatch.com list using feedBase, and do not want to have to invent a new Twitter user just for that purpose. That's the basic use-case. I wanted to get it reliable and debugged at this level, and then grow it out. Glad you find it interesting!
